code/src/java/pcgen/util/PJEP.java

```diff
@@ -301,7 +301,7 @@ else if (parent instanceof PlayerCharacter)
 	 * @return Returns the variableSource.
 	 */
 	@Deprecated
-	protected String getVariableSource()
+	private String getVariableSource()
```

Comment:
Unfortunately it's used, and since this is the old formula system, I think it's better off not being messed with, as we're phasing it out anyway.